docs/SOP.md

```diff
@@ -100,6 +101,20 @@ In general, the metadata record of an ontology in the OBO Foundry metadata regis
 
 For a discussion on this SOP, see [here](https://github.com/OBOFoundry/OBOFoundry.github.io/issues/1848).
 
+<a name="REACTIVATION"></a>
+
+## Reactivating obsolete, orphaned, or inactive ontologies
+
+To mark ontologies that were marked as either obsolete, orphaned, or inactive as active again, we follow this process. 
+Currently, these are the required steps:
+
+1. A member of the development team of the ontology makes a pull request (PR) on the metadata file with the desired change in status. The PR must include evidence to demonstrate new activity, for example by referring to recently-closed issues (we will provide more detailed guidelines as this SOP matures).
```

Comment:
A person involved in developing the ontology (usually the contact person) makes a pull request (PR) on the metadata file with the desired change in status. The PR must include evidence to demonstrate new activity, for example by referring to recently-closed issues (we will provide more detailed guidelines as this SOP matures). If the PR was made by someone other than the contact person, or if the contact person has changed, the contact should be tagged in the PR so that the proper followup can be done. The contact person then has one month to verify that the change is desired.

Reply:
You should learn suggestion mode! Click on the most left button on any comment:

<img width="861" alt="image" src="https://user-images.githubusercontent.com/7070631/226925721-7a4a8fd5-5352-4d91-bebc-d8a8e5658e22.png">